src/main/java/org/overture/ego/controller/AuthController.java

```diff
@@ -92,15 +97,19 @@ boolean verifyJWToken(
   private String generateUserToken(Map authInfo){
     val userName = authInfo.get("email").toString();
     User user = userService.getByName(userName, false);
+
     if (user == null) {
       user = createNewUser(userName,authInfo);
-      userService.create(user);
+      if (saveUsersAllowed) {
```

Comment:
We'll need to do it for facebooktokens as well

Reply:
Controlling user creation at this point should work for facebook and google - This is in the `generateUserToken(...)` function called by both `exchangeFacebookTokenForAuth(...)` and `exchangeGoogleTokenForAuth(...)` .